src/v2/Components/Pagination/index.tsx

```diff
@@ -1,18 +1,17 @@
 import React from "react"
 import { createFragmentContainer, graphql } from "react-relay"
 import { Media } from "v2/Utils/Responsive"
-import { ScrollIntoView } from "v2/Utils/ScrollIntoView"
```

Comment:
In general this component seems like a bad idea. It was the case that if you clicked *anywhere* on the pagination component it'd scroll to the top. So you could click and miss the link and it'd just push up back up. And of course navigating via keyboard this wouldn't work.